Beams/MRML/vtkMRMLRTPlanNode.h

```diff
@@ -221,6 +237,9 @@ class VTK_SLICER_BEAMS_MODULE_MRML_EXPORT vtkMRMLRTPlanNode : public vtkMRMLNode
   ///TODO: Allow user to specify dose volume resolution different from reference volume
   /// (currently output dose volume has the same spacing as the reference anatomy)
   double DoseGrid[3];
+
+  /// Flag for ion plan

```

Comment:
Can you be more specific? If the flag means that the plan is an ion plan, then please write this.